src/authzedapi/authzed/api/v0/Namespace_serviceServiceClientPb.ts

```diff
@@ -0,0 +1,119 @@
+/**
```

Comment:
can we add a .gitattributes entry to mark these as generated?

Reply:
Done